### DISCUSSION TITLE
OpenSnitch 1.5.8-1 not allowing outbound traffic

### DISCUSSION THREAD

**Discussion by Brian67** [evilsocket/opensnitch#923] [General]
I am rather new to Linux and I could use a little help with troubleshooting.

Using Manjaro Linux and kernel version 6.1.25-1.  I have used OpenSnitch on older versions of Manjaro with complete success.  But now it is not actually letting any outbound traffic, even though the events show outbound traffic activity.  My rules are solid, they are definitely configured to Allow-Always for things like my browser, WhatsDesk, PAMAC, et al.

Is OpenSnitch 1.5.8-1 compatible with Kernel 6.1.25-1?

Any idea why this may be happening?

**Comment by gustavo-iniguez-goya** [comment]
Hi @Brian67 ,

Yes, it should be compatible with kernel 6.x. Did you upgrade to version 1.5.8 recently?

Please, set LogLevel to DEBUG under Preferences -> Nodes, connect to a host domain with your web browser (https://www.reddit.com for example) and to an IP (for example https://1.1.1.1),  and post the log file `/var/log/opensnitchd.log`
Also let us know if the connection to the IP works.

On the other hand, if you use systemd-resolved, could you restart it and try again? `$ sudo service systemd-resolved restart`

In my system systemd-resolved stops working from time to time, and only restarting it restores the connectivity.
